app/domain/authentication/authn_jwt/consts.rb

```diff
@@ -20,11 +20,15 @@ module AuthnJwt
     EXP_CLAIM_NAME = "exp"
     NBF_CLAIM_NAME = "nbf"
     IAT_CLAIM_NAME = "iat"
+    JTI_CLAIM_NAME = "jti"
     SUPPORTED_ALGORITHMS = %w[RS256 RS384 RS512].freeze
     CACHE_REFRESHES_PER_INTERVAL = 10
     CACHE_RATE_LIMIT_INTERVAL = 300
     CACHE_MAX_CONCURRENT_REQUESTS = 3
     MANDATORY_CLAIMS = [EXP_CLAIM_NAME].freeze
     OPTIONAL_CLAIMS = [ISS_CLAIM_NAME, NBF_CLAIM_NAME, IAT_CLAIM_NAME].freeze
+    VALID_CLAIM_NAME_REGEX = /^[a-zA-Z|$|_][a-zA-Z|$|_|0-9]*$/ # starts with letter $ or _ can contains digits
+    MANDATORY_CLAIMS_RESOURCE_NAME = "mandatory-claims"
+    MANDATORY_CLAIMS_DENY_LIST = [ISS_CLAIM_NAME, EXP_CLAIM_NAME, NBF_CLAIM_NAME, IAT_CLAIM_NAME, JTI_CLAIM_NAME].freeze
```

Comment:
why do we work with deny list instead of allow list for the allowed claims?

Reply:
because this is custom claim so you can configure what ever you want except of the JWT reserved claims